src/components/footer/Footer.js

```diff
@@ -1,95 +1,209 @@
-import { Box, useColorModeValue, Grid, GridItem, Link, Text, Flex } from '@chakra-ui/react';
+import { Box, useColorModeValue, Grid, GridItem, Link, Text, Flex, useMediaQuery } from '@chakra-ui/react';
 import LensBoosterIcon from '../icons/LensBoosterIcon';
 import TwitterFooter from '../icons/TwitterFooter';
 import DiscordFooter from '../icons/DiscordFooter';
 import FiveElementsFooter from '../icons/FiveElementsFooter';
 import ContactFooter from '../icons/ContactFooter';
 import LensFooter from '../icons/LensFooter';
+import MediumFooter from '../icons/MediumFooter';
+import DocsFooter from '../icons/DocsFooter';
 
 export default function Footer() {
+  const [isLargerThan640] = useMediaQuery('(min-width: 640px)');
   return (
     <Box backgroundColor={useColorModeValue('light_azure', 'light_azure')} padding={7}>
-      <Grid
-        templateRows="repeat(1, 1fr)"
-        templateColumns="repeat(12, 1fr)"
-        maxW="container.xl"
-        mx="auto"
-        alignItems="center"
-        fontFamily="'Prompt', sans-serif"
-        fontWeight={600}
-        gap={{ base: '25px', md: 0 }}
-      >
-        <GridItem colSpan={{ base: '6', md: '2' }} display={{ base: 'none', md: 'inherit' }} alignContent>
-          {' '}
-          <LensBoosterIcon width="20rem" />
-        </GridItem>
-        <GridItem colSpan={{ base: '6', md: '2' }}>
-          {' '}
-          <Link href="https://discord.gg/bxfTM37Xyk">
-            <Flex>
-              <Box w="fit-content" my="auto" mr={2}>
-                <ContactFooter />
-              </Box>{' '}
-              <Text color="#5C6F81" width="fit-content" display="inline" fontSize={12}>
-                Contact us
-              </Text>
-            </Flex>
-          </Link>
-        </GridItem>
+      {isLargerThan640 && (
+        <Grid
+          templateRows="repeat(1, 1fr)"
+          templateColumns="repeat(12, 1fr)"
+          w="90%"
+          maxW="container.2xl"
+          mx="auto"
+          alignItems="center"
+          fontFamily="'Prompt', sans-serif"
+          fontWeight={600}
+          gap={{ base: '25px', md: 0 }}
+        >
+          <GridItem colSpan={{ base: '4', md: '1' }} display={{ base: 'none', md: 'inherit' }} alignContent>
+            {' '}
+            <LensBoosterIcon />
+          </GridItem>
+          <GridItem colSpan={{ base: '6', md: '3' }}>
+            {' '}
+            <Link href="https://fiveelementslabs.com/">
+              <Flex>
+                <Box w="fit-content" my="auto" mr={2}>
+                  <FiveElementsFooter />
+                </Box>{' '}
+                <Text color="#5C6F81" width="fit-content" display="inline" fontSize={15}>
+                  By Five Elements Labs
+                </Text>
+              </Flex>
+            </Link>
+          </GridItem>
+          <GridItem colSpan={{ base: '6', md: '2' }}>
```

Comment:
Can't we add this `GridItem` to the Theme?

Reply:
chakra can't do that